photon-ml/src/integTest/scala/com/linkedin/photon/ml/function/ObjectiveFunctionIntegTest.scala

```diff
@@ -309,31 +285,36 @@ class ObjectiveFunctionIntegTest extends SparkTestUtils {
    * In the near term, we should decide if we want to keep those methods as part of the
    * design or remove them, as they aren't used by any of the solvers.
    */
-  @Test(dataProvider = "getDifferentiableFunctions",
-    groups = Array[String]("ObjectiveFunctionTests", "testCore"))
-  def checkGradientConsistentWithObjectiveSpark(description: String, function: DiffFunction[LabeledPoint], localData: Seq[LabeledPoint], treeAggregateDepth: Int): Unit = sparkTest("checkGradientConsistentWithObjectiveSpark") {
-    function.treeAggregateDepth = treeAggregateDepth
-    val data = sc.parallelize(localData).repartition(Runtime.getRuntime.availableProcessors)
-    val r: Random = new Random(ObjectiveFunctionIntegTest.PARAMETER_RANDOM_SEED)
+  @Test(dataProvider = "getDifferentiableFunctions", groups = Array[String]("ObjectiveFunctionTests", "testCore"))
+  def checkGradientConsistentWithObjectiveSpark(description: String, function: DiffFunction[LabeledPoint],
+      localData: Seq[LabeledPoint], treeAggregateDepth: Int): Unit = sparkTest(
+    "checkGradientConsistentWithObjectiveSpark") {
```

Comment:
If all constructor arguments cannot fit on one line, then each argument should be on its own line.
http://docs.scala-lang.org/style/declarations.html